web/src/index.js

```diff
@@ -1,15 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { createHashRouter, RouterProvider } from "react-router-dom";
-import About from "components/about/about";
+import About from "components/about/About";
```

Comment:
merge error? should be lowercase

Reply:
yeah, there was some error about these that was coming through. I'll fix these